scroll-search.lic

```diff
@@ -0,0 +1,520 @@
+=begin
+  Documentation: TBD
+=end
+
+custom_require.call(%w(common common-items drinfomon))
+
+# Define Scroll Info class
+class ScrollInfo
+  def initialize(container, index, container_index, ordinal, spell, type)
+    # Instance variables
+    @container = container
+    @index = index
+	@container_index = container_index
+    @ordinal = ordinal
+    @spell = spell
+    @type = type
+  end
+
+  def get_container
```

Comment:
You can just make all of these accessors so people can just do scroll_info.index scroll_info.container etc. Name things after what they return not what they do.